### ISSUE TITLE
Error when try to init grafana dashboard

### ISSUE THREAD

**Issue by y0an** [pyaillet/prometheus-travel#7]
```
bash-4.4# ./run.sh 
Preparing data import
 ...
importing /opt/grafana-dashboards/delorean-dashboard.json
curl: (22) The requested URL returned error: 500 Internal Server Error

```

**Comment by pyaillet** [comment]
The export format isn't exactly the same as the api one...
I'll export the dashboard via the api and push it here :)

**Comment by pyaillet** [comment]
Closed by b34c980
